### ISSUE TITLE
There may be problems in the documentation of batchnorm

### ISSUE THREAD

**Issue by gyp2448565528** [pytorch/pytorch#82313]
### 📚 The doc issue

The documentation about Batchnorm1d says ‘The standard-deviation is calculated via the biased estimator, equivalent to torch.var(input, unbiased=False).’
However, the experimental results are contrary to the documentation.
![屏幕截图 2022-07-27 213357](https://user-images.githubusercontent.com/33481094/181260063-e9144d3f-c009-4f6b-8cae-58c505b6389d.png)
![屏幕截图 2022-07-27 213301](https://user-images.githubusercontent.com/33481094/181259864-acd7b8ce-e72e-4612-bd1c-a05dc8bdf9b4.png)

cc @svekars @holly1238 @albanD @mruberry @jbschlosser @walterddr @kshitij12345 @saketh-are

**Comment by jbschlosser** [comment]
Duplicate of #77427? As mentioned there, we'd accept a PR clarifying the situation in the docs if you'd like to contribute!
